src/clientEntry.tsx

```diff
@@ -6,15 +6,20 @@ import { HelmetProvider } from 'react-helmet-async'
 import { BrowserRouter } from 'react-router-dom'
 import { HotApp } from './App'
 import { apolloClient } from './client/apolloClient'
+import { PageTracker } from './components/PageTracker'
 import { createSession } from './utils/sessionStorage'
 
 ReactDOM.hydrate(
   <BrowserRouter>
-    <HelmetProvider>
-      <ApolloProvider client={apolloClient!.client!}>
-        <HotApp session={createSession(new CookieStorage({ expires: null }))} />
-      </ApolloProvider>
-    </HelmetProvider>
+    <PageTracker>
```

Comment:
Don't we wanna add this for each route? Otherwise it's just gonna track the first page load right?

Reply:
PageTracker actually hooks itself up to `history` and emits an `analytics.page()`-call each time the route changes. :+1:

Reply:
right 😅too early to do PRs